src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl() {
+        };
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User("user", "111111", 22);
+    }
+
+    @Test
+    void inputValidData_Ok() {
```

Comment:
<img width="887" alt="Screenshot 2022-05-07 at 20 01 43" src="https://user-images.githubusercontent.com/60258373/167264417-ba329b3b-1c16-4aba-a4a3-808134eed62b.png">
the same for other methods